google/cloud/bigquery_v2/gapic/enums.py

```diff
@@ -0,0 +1,171 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrappers for protocol buffer enum types."""
+
+import enum
+
+
+class Model(object):
+    class DataSplitMethod(enum.IntEnum):
+        """
+        Indicates the method to split input data into multiple tables.
+
+        Attributes:
+          DATA_SPLIT_METHOD_UNSPECIFIED (int)
+          RANDOM (int): Splits data randomly.
+          CUSTOM (int): Splits data with the user provided tags.
+          SEQUENTIAL (int): Splits data sequentially.
+          NO_SPLIT (int): Data split will be skipped.
+          AUTO_SPLIT (int): Splits data automatically: Uses NO_SPLIT if the data size is small.
+          Otherwise uses RANDOM.
+        """
+
+        DATA_SPLIT_METHOD_UNSPECIFIED = 0
+        RANDOM = 1
+        CUSTOM = 2
+        SEQUENTIAL = 3
+        NO_SPLIT = 4
+        AUTO_SPLIT = 5
+
+    class DistanceType(enum.IntEnum):
+        """
+        Distance metric used to compute the distance between two points.
+
+        Attributes:
+          DISTANCE_TYPE_UNSPECIFIED (int)
+          EUCLIDEAN (int): Eculidean distance.
+          COSINE (int): Cosine distance.
+        """
+
+        DISTANCE_TYPE_UNSPECIFIED = 0
+        EUCLIDEAN = 1
+        COSINE = 2
+
+    class LearnRateStrategy(enum.IntEnum):
+        """
+        Indicates the learning rate optimization strategy to use.
+
+        Attributes:
+          LEARN_RATE_STRATEGY_UNSPECIFIED (int)
+          LINE_SEARCH (int): Use line search to determine learning rate.
+          CONSTANT (int): Use a constant learning rate.
+        """
+
+        LEARN_RATE_STRATEGY_UNSPECIFIED = 0
+        LINE_SEARCH = 1
+        CONSTANT = 2
+
+    class LossType(enum.IntEnum):
+        """
+        Loss metric to evaluate model training performance.
+
+        Attributes:
+          LOSS_TYPE_UNSPECIFIED (int)
+          MEAN_SQUARED_LOSS (int): Mean squared loss, used for linear regression.
+          MEAN_LOG_LOSS (int): Mean log loss, used for logistic regression.
+        """
+
+        LOSS_TYPE_UNSPECIFIED = 0
+        MEAN_SQUARED_LOSS = 1
+        MEAN_LOG_LOSS = 2
+
+    class ModelType(enum.IntEnum):
+        """
+        Indicates the type of the Model.
+
+        Attributes:
+          MODEL_TYPE_UNSPECIFIED (int)
+          LINEAR_REGRESSION (int): Linear regression model.
+          LOGISTIC_REGRESSION (int): Logistic regression based classification model.
+          KMEANS (int): K-means clustering model.
+          TENSORFLOW (int): [Beta] An imported TensorFlow model.
+        """
+
+        MODEL_TYPE_UNSPECIFIED = 0
+        LINEAR_REGRESSION = 1
+        LOGISTIC_REGRESSION = 2
+        KMEANS = 3
+        TENSORFLOW = 6
```

Comment:
Still not seeing `AUTOML_REGRESSOR` here

https://github.com/googleapis/python-bigquery/issues/293#issuecomment-703653854